packages/core/src/pseudo-states.ts

```diff
@@ -295,6 +295,14 @@ function resolveStateValue(
         node.content || stateDef.defaultValue
     );
 
+    if (!actualParam && rule) {
+        diagnostics.warn(
+            rule,
+            `pseudo-state "${name}" expects a parameter of type ${stateDef.type} but none was given`,
```

Comment:
Add the warning (maybe error?) using the "state Errors" also use the same error function in the tests

Reply:
Not sure I fully understand. Make it an error sounds legit tho.